iis-wf/iis-wf-affmatching/src/main/java/eu/dnetlib/iis/wf/affmatching/model/DocumentProject.java

```diff
@@ -0,0 +1,71 @@
+package eu.dnetlib.iis.wf.affmatching.model;
+
+import org.apache.commons.lang3.StringUtils;
+
+import com.google.common.base.Preconditions;
+
+/**
+ * Paired document and project identifiers.
+ * 
+ * Confidence level describes the level of confidence given relation is valid.
+ * 
+ * @author mhorst
+ *
+ */
+public class DocumentProject {
+
+    private String documentId;
+
+    private String projectId;
+
+    private Float confidenceLevel;
+
+    // ------------------------ CONSTRUCTORS --------------------------
+
+    public DocumentProject(String documentId, String projectId, Float confidenceLevel) {
+        Preconditions.checkArgument(StringUtils.isNotBlank(documentId));
+        Preconditions.checkArgument(StringUtils.isNotBlank(projectId));
+        Preconditions.checkArgument(confidenceLevel != null && confidenceLevel >= 0 && confidenceLevel <= 1);
```

Comment:
Well, I don't understand. If the confidenceLevel may not be null then maybe it should be float (not Float).